examples/Intermediate-parameters.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
the filename should start with a lowercase (it is currently capitalized) following the convention of all other examples in this folder

Reply:
Thank you. Fixed.